alf/utils/common_test.py

```diff
@@ -23,15 +23,16 @@ def test_transform_image(self):
         observation = tf.zeros(shape, dtype=tf.uint8)
         common.image_scale_transformer(observation)
 
-        T1 = common.namedtuple('T1', ['x', 'y'])
+        T1 = common.namedtuple('T1', ['x'])
         T2 = common.namedtuple('T2', ['a', 'b', 'c'])
-        T3 = common.namedtuple('T3', ['l', 'm'])
+        T3 = common.namedtuple('T3', ['l'])
```

Comment:
Should not remove 'm' because it's used below: 
```
with self.assertRaises(Exception) as _:	    
            common.image_scale_transformer(
                observation, fields=["x.b.m"])  # empty ()
```

Reply:
reverted